teams/nixos_release.tt

```diff
@@ -0,0 +1,41 @@
+[% WRAPPER layout.tt hideTitle=1 title="NixOS Release Team" menu='nixos' %]
+
+<h1>NixOS Release Team</h1>
+
+<div class="lead">
+  For each release there are two release managers. After each release the
+  release manager having managed two releases steps down and the release
+  management team of the last release appoints a new release manager.
+</div>
+
+<p>
+  This makes sure a release management team always consists of one release
+  manager who already has managed one release and one release manager being
+  introduced to their role, making it easier to pass on knowledge and
+  experience.
+</p>
+
+<h2>Members</h2>
+
+<ul>
+  <li><a href="https://github.com/worldofpeace">@worldofpeace</a></li>
+  <li><a href="https://github.com/disassembler">@disassembler</a></li>
+</ul>
```

Comment:
I think it's important to specify for what release the manager has served primarily?
So I'm (20.03) and Samuel is (19.09).